server/graphql/v2/query/collection/AccountsQuery.ts

```diff
@@ -65,6 +65,7 @@ const AccountsQuery = {
         onlyActive: args.isActive ? true : null,
         skipRecentAccounts: args.skipRecentAccounts,
         hasCustomContributionsEnabled: args.hasCustomContributionsEnabled,
+        countries: args.countries,
```

Comment:
I don't see `args.countries` being defined in `args`

Reply:
Also, according to https://github.com/opencollective/opencollective/issues/4378, shouldn't that be `country`?

Reply:
> I don't see `args.countries` being defined in `args`

Thanks Ben. This is due to migrating to GQL v2 and the front-end query still using graphql v1. Anyways, I've opened a new PR so that the country filter uses GQL v2; https://github.com/opencollective/opencollective-api/pull/7363

> Also, according to [opencollective/opencollective#4378](https://github.com/opencollective/opencollective/issues/4378), shouldn't that be `country`?

Good point, I've missed this. This was corrected in the aforementioned PR as well.